sunpy/time/time.py

```diff
@@ -273,24 +273,11 @@ def is_time(time_string, time_format=None):
     out : bool
         True if can be parsed by parse_time
 
-    Notes
-    -----
-        If time_string is an instance of float, then it is assumed to be in
-        unix time format.
```

Comment:
What is the case now?

Reply:
You have to specify the format yourself with the `format` keyword.